osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSubScreen.cs

```diff
@@ -49,6 +49,9 @@ public partial class MultiplayerMatchSubScreen : RoomSubScreen, IHandlePresentBe
         [Resolved]
         private MultiplayerClient client { get; set; }
 
+        [Resolved]
```

Comment:
```suggestion
        [Resolved(canBeNull: true)]
```

to fix existing tests at least